pages/index.vue

```diff
@@ -1,42 +1,53 @@
 <template>
 
-	<b-container>
+	<b-container fluid>
 
-		<h1>Origami Annotation Tool</h1>
+		<!-- Navigation bar -->
+		<tool-navbar :navItemsState="navItemsState" :pageName="fullName"></tool-navbar>
 
-		<!-- Debug component - shows file contents -->
+		<!-- TSV file loading area -->
 		<b-row>
 			<h2>TSV File</h2>
-			<textarea rows="5" cols="200" v-model="stringifiedTsvFile"></textarea>
-		</b-row>		
 
-		<!-- Selects participant.tsv file -->
-		<file-selector 
-			content-type="text/tab-separated-values"
-			v-on:file-selected="saveTsvFileData($event)">
-		</file-selector>
+			<!-- Debug component - shows file contents -->			
+			<textarea rows="5" cols="200" v-model="stringifiedTsvFile" :key="currentState"></textarea>
+			
+			<!-- Selects participant.tsv file -->
+			<file-selector 
+				content-type="text/tab-separated-values"
```

Comment:
as far as I can see, the content type is not validated by this. Is there an easy way to validate that folks actually uploaded the right file type?

Reply:
The input element within the file-selector will ensure that at least the file extension matches the requested content type. If we want to validate the contents, we have to do that ourselves – or via a third party library.

Reply:
Added an issue for this: https://github.com/metaneuro/annotation_tool/issues/49